backbone.js

```diff
@@ -394,6 +394,7 @@
   var Model = Backbone.Model = function(attributes, options) {
     var attrs = attributes || {};
     options || (options = {});
+    _.extend(this, _.result(this, 'localProps'));
```

Comment:
My thought was around the secondary use case (in my mind) of adding properties to the instance.  I was thinking a user might want to do that with a few static properties.  But I'm realizing now that the reference would be shared anyway and that would undermine the purpose. So I'm on board with that.